framework/src/org/checkerframework/framework/util/AnnotationBuilder.java

```diff
@@ -147,6 +148,25 @@ public String toString() {
         };
     }
 
+    /**
+     * Copies every element value from the given annotation to the annotation to be built if the
+     * element name match
+     */
+    public AnnotationBuilder copyElementValuesFromAnnotation(AnnotationMirror annotation) {
+        for (Map.Entry<? extends ExecutableElement, ? extends AnnotationValue> entry :
```

Comment:
I would pick clearer names for the variables to distinguish better which ones are input and which ones are modified.